plugins/processors/t128_rate/README.md

```diff
@@ -0,0 +1,36 @@
+# T128 Rate Processor Plugin
+
+The `t128_rate` processor computes the point to point difference of a field.
+
+### Configuration:
+
+```toml
+[[processors.t128_rate]]
+  ## If more than this amount of time passes between data points, the
+  ## previous value will be considered old and the rate will be recalculated
+  ## as if it hadn't been seen before. A zero expiration means never expire.
+  # expiration = "0s"
+
+  ## For the fields who's key/value pairs don't match, should the original
+  ## field be removed?
+  # remove-original = true
+
+[processors.t128_rate.fields]
+  ## Replace fields with their rates, renaming them if indicated
+  # "/rate/metric" = "/total/metric"
+  # "/inline/replace" = "/inline/replace"]
+```
+
+### Example processing:
+
+```toml
+[[processors.t128_rate]]
+[processors.t128_rate.fields]
+  rate = total
+```
+
+```diff
+- measurement total=10i 1502489900000000000
+- measurement total=15i 1502490000000000000
++ measurement rate=5i 1502490000000000000
```

Comment:
I'm slightly confused by this example - are we computing rate as in `delta / time` or just `delta`? This example would imply the latter but I thought we need the former.

Reply:
Yeah, it ends up just being the delta based on how we store/retrieve in the backend.